subt/main.py

```diff
@@ -935,6 +935,12 @@ def main():
         app.play()
 
     elif args.command == 'run':
+        import logging
+        logging.basicConfig(
+            level=logging.DEBUG,
```

Comment:
I am not sure if we want always debug logging level (including real robots), but ... OK. If it will cause troubles we may add it as command line argument ...